libs/langgraph/tests/test_state.py

```diff
@@ -318,3 +319,38 @@ def class_method(self, state):
 
     # class method
     assert _get_node_name(MyClass().class_method) == "class_method"
+
+
+def test_input_schema_conditional_edge():
```

Comment:
this tests a related issue, will add another explicit test for different schema in conditional edge